contents/blog/changing-to-self-serve.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2022-05-19
+title: Next Up - Nailing Self Serve
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/ceo-diary.png
+featuredImageType: full
+---
+
+The longer your strategy gets, the more useless it becomes.
+
+At PostHog, we therefore communicate strategy as 'Nail X', where X is the thing we need to nail. It's simple and effective.
+
+We [recently switched](https://github.com/PostHog/posthog.com/pull/3395) from Nail Enterprise to Nail Self Serve. This walks you through the previous context, and why we made that change.
+
+## Why did we focus on Nail Enterprise in the first place?
+
+The best products are generally built for end users.
+
+The game we're playing isn't to please C-level executives or procurement - we want an engineer in any size organization on Earth to be able to use our software in production without getting up from their desk. If they work at a larger company, and if we provide them with a great experience, they'll share the platform with their peers, and will probably want to upgrade to one of our [paid offerings](/pricing). Our open source and free user base is our sales team.
+
+Our original go to market plan was therefore a little like this:
+
+1. get our open source project to product market fit
+1. get a paid mid-size (at the time, ~$20K-$40K a year) product to product market fit
+1. get a paid large-size ($200K+ a year) product to product market fit
```

Comment:
Assume this is meant to be a 1,2,3 list, or is this deliberate?

Reply:
this should turn into a 1,2,3 list automatically - check the preview